README.md

```diff
@@ -1,227 +1,33 @@
-# WebAssembly for Proxies (C++ SDK)
+# Proxy-Wasm C++ SDK
```

Comment:
Nit: All REAMDEs use "WebAssembly for Proxies (xxx)" format, so it might make sense to leave this as is... or we make this change everywhere.

Reply:
Done.